src/Query/QueryInterface.php

```diff
@@ -249,10 +249,10 @@ public function getWithQueries(): array;
      * @throws InvalidConfigException
      * @throws Throwable
      *
-     * @return array|null The first row (in terms of an array) of the query result. It returns `null` if the query
-     * results in nothing.
+     * @return array|object|null The first row (as array or object) of the query result.
+     * It returns `null` if the query results in nothing.
      */
-    public function one(): array|null;
+    public function one(): array|object|null;
```

Comment:
Should we make same change in `all()` method and in `Query` class?

Reply:
fixed